src/borg/crypto/key.py

```diff
@@ -417,6 +417,7 @@ def init_ciphers(self, manifest_data=None):
 
 class FlexiKey:
     FILE_ID = "BORG_KEY"
+    STORAGE = KeyBlobStorage.NO_STORAGE  # override in subclass
```

Comment:
```suggestion
    STORAGE: ClassVar[KeyBlobStorage] = KeyBlobStorage.NO_STORAGE  # override in subclass
```
for extra clarity

Reply:
hmm, `KeyBlobStorage.NO_STORAGE` is a str.

Reply:
:facepalm: my mistake, usage looked like a enum

Reply:
see 2705ccb